logger/plotter.py

```diff
@@ -91,15 +102,19 @@ def plot_metric(self, metric):
         name, tag = metric.name, metric.tag
         cache = self.cache[metric.name_id()]
         cache.update(metric)
-        sent = self._plot_xy(name, tag, cache.x, cache.y,
-                             metric.time_idx)
+        sent = self._plot_xy(name, tag, cache.x, cache.y, metric.time_idx)
         # clear cache if data has been sent successfully
         if sent:
             cache.clear()
 
     def plot_config(self, config):
+        config = dict((str(k), v) for (k, v) in config.items())
```

Comment:
Why do you take the representation just for the key and not the value?

Reply:
The `str` is just to avoid unnecessary distinctions between unicode, raw string literals etc. I don't think that's needed for the value